module2/homework/transformContainers/transform.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <deque>
+#include <list>
+#include <map>
+#include <string>
```

Comment:
Header includes and standard library includes should be in the another paragraphs.